control-plane-operator/controllers/hostedcontrolplane/kas/params.go

```diff
@@ -477,7 +478,8 @@ func NewKubeAPIServerServiceParams(hcp *hyperv1.HostedControlPlane) *KubeAPIServ
 		port = int(*hcp.Spec.APIPort)
 	}
 	return &KubeAPIServerServiceParams{
-		APIServerPort:  port,
-		OwnerReference: config.ControllerOwnerRef(hcp),
+		APIServerPort:     port,
```

Comment:
It would be nice to have some unittest that checks that setting this on the hcluster actually results in it being set on the service

Reply:
At the `(r *HostedControlPlaneReconciler) reconcileAPIServerService` level, or the `kas.ReconcileService()` level?

Reply:
At the reconcileAPIServerService seems fine to me